lightning/src/routing/mod.rs

```diff
@@ -53,3 +56,10 @@ impl<S: Score> LockableScore<S> {
 		self.scorer.lock().unwrap()
 	}
 }
+
+impl<S: Score> Writeable for LockableScore<S> {
```

Comment:
Hmm, this seems like something we might want upstream?

Reply:
Yea, I think so, eventually at least. Either way we need it for 0.0.103 bindings, so gotta have it here.